aineko/extras/connectors/rest_poller.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for connecting to a REST endpoint."""
+
+import json
+import time
+from typing import Any, Dict, List, Optional
+
+import requests
+from pydantic import BaseModel, field_validator
+
+from aineko import AbstractNode
+
+
+class ParamsRESTPoller(BaseModel):
+    """Connector params for REST model."""
+
+    timeout: int = 10
+    url: str
+    headers: Optional[Dict[str, Any]] = None
+    data: Optional[Dict[str, Any]] = None
+    poll_interval: float = 5.0
+    max_retries: int = -1
+    metadata: Optional[Dict[str, Any]] = None
+    retry_sleep: float = 5
+    success_codes: List[int] = [
+        200,
+        201,
+        202,
+        203,
+        204,
+        205,
+        206,
+        207,
+        208,
+        226,
+    ]
+
+    @field_validator("url")
+    @classmethod
+    def supported_url(cls, url: str) -> str:
+        """Validates that the url is a valid HTTP or HTTPS URL."""
+        if not (url.startswith("https://") or url.startswith("http://")):
+            raise ValueError(
+                "Invalid url provided to HTTPS params. "
+                'Expected url to start with "https://". '
+                f"Provided url was: {url}"
+            )
+        return url
+
+
+# pylint: disable=anomalous-backslash-in-string
+class RESTPoller(AbstractNode):
+    """Connects to an REST endpoint via HTTP or HTTPS and polls.
+
+    This node is a wrapper around the
+    [requests](
+        https://docs.python-requests.org/en/master/
+        ){:target="\_blank"} library.
```

Comment:
```suggestion
class RESTPoller(AbstractNode):
    """Connects to an REST endpoint via HTTP or HTTPS and polls.

    This node is a wrapper around the
    [requests](
        https://docs.python-requests.org/en/master/
        ){:target="_blank"} library.
```
I don't think this backslash is needed to get the expected behavior. See PR #141